xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -179,6 +179,23 @@ static int PlayerControl(const std::vector<std::string>& params)
       g_application.GetAppPlayer().SetTempo(playTempo);
     }
   }
+  else if (StringUtils::StartsWithNoCase(params[0], "tempo"))
+  {
+    if (params[0].size() == 5)
+      CLog::Log(LOGERROR, "PlayerControl(tempo(n)) called with no argument");
+    else if (params[0].size() < 8) // arg must be at least "(N)"
+      CLog::Log(LOGERROR, "PlayerControl(tempo(n)) called with invalid argument: \"%s\"",
+                params[0].substr(6).c_str());
+    else if (g_application.GetAppPlayer().SupportsTempo() &&
+             g_application.GetAppPlayer().IsPlaying() && !g_application.GetAppPlayer().IsPaused())
+    {
+      std::string strTempo = params[0].substr(6);
+      StringUtils::TrimRight(strTempo, ")");
+      float playTempo = static_cast<float>(atof(strTempo.c_str()));
```

Comment:
Since c++11, there is strtof => https://en.cppreference.com/w/cpp/string/byte/strtof. No need for the cast, then.

Reply:
thx for the hint. i've updated it now. could you please double-check?